packages/pwa-buildpack/lib/Utilities/__tests__/createFileHash.spec.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const fs = require('fs');
+
+const createFileHash = require('../createFileHash');
+
+const DEFAULT_SAMPLE_FILE_PATH = path.join(
+    path.resolve(__dirname).split('/packages')[0],
+    'packages',
+    'pwa-buildpack',
+    'lib',
+    'Utilities',
+    '__tests__',
+    'sampleFile.txt'
+);
+
+test('Should return a promise that resolves to a string', async () => {
+    const returnValue = createFileHash(DEFAULT_SAMPLE_FILE_PATH);
+
+    expect(returnValue instanceof Promise).toBeTruthy();
+
+    const hash = await returnValue;
+
+    expect(typeof hash).toBe('string');
+});
+
+test('Should return same hash if the contents of the file has not changed', async () => {
+    const oldValue = await createFileHash(DEFAULT_SAMPLE_FILE_PATH);
+    const newValue = await createFileHash(DEFAULT_SAMPLE_FILE_PATH);
+
+    expect(oldValue === newValue).toBeTruthy();
```

Comment:
Check out the [toBe assertion](https://jestjs.io/docs/en/expect#tobevalue) for this one.

Reply:
Not sure how it missed that 👍